modules/private-cluster/variables.tf

```diff
@@ -366,6 +366,13 @@ variable "master_ipv4_cidr_block" {
   default     = "10.0.0.0/28"
 }
 
+variable "master_global_access_enabled" {
+  type        = bool
+  description = "Whether the cluster master is accessible globally (from any region) or only within the same region as the private endpoint."
+
+  default = true
```

Comment:
@ericyz can we default this to false for now and open an issue to flip this to true in a following breaking release. This will allow us to put this in a minor release for now.

Reply:
Sure.

Reply:
@ericyz FYI I switched it back since next release is breaking https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/pull/1421/commits/731e5ec00b7e88ad5651a355f220ee98bdce3988